coda/coda_validate/models.py

```diff
@@ -1,7 +1,36 @@
 from django.db import models
 from datetime import datetime
 
-# Create your models here.
+
+class ValidateManager(models.Manager):
+
+    def last_verified_status_counts(self):
+        results = self._verified_counts()
+        return VerifiedCountsResultFormatter(results).format()
+
+    def _verified_counts(self):
+        return (self.values('last_verified_status')
+                    .annotate(count=models.Count('last_verified_status')))
```

Comment:
In running coda to look at #95, I am noticing unexpected results being returned from this aggregation.
```
[{'count': 1, 'last_verified_status': u'Unverified'}, {'count': 1, 'last_verified_status': u'Unverified'}, {'count': 1, 'last_verified_status': u'Passed'}, {'count': 1, 'last_verified_status': u'Unverified'}, {'count': 1, 'last_verified_status': u'Failed'}, {'count': 1, 'last_verified_status': u'Failed'}]
```
Explicitly getting rid of the query ordering using `order_by()` is fixing the issue for me:
```
         return (self.values('last_verified_status')
                    .annotate(count=models.Count('last_verified_status')).order_by())
```
Has anyone else noticed this behavior?